app/Jobs/ProcessSubmission.php

```diff
@@ -242,10 +248,11 @@ private function getRemoteSubmissionFileHandle($filename)
         rename($_t, $tmpFilename);
 
         $client = new \GuzzleHttp\Client();
-        $response = $client->request('GET',
-            config('app.url') . '/api/v1/getSubmissionFile.php',
-            ['query' => ['filename' => $filename],
-                  'sink' => $tmpFilename]);
+        $url = config('app.url') . '/api/v1/getSubmissionFile.php';
```

Comment:
Using the `url()` helper would be cleaner:
```suggestion
        $url = url('/api/v1/getSubmissionFile.php');
```